packages/unchained-client/src/evm/ethereum/parser/yearn.ts

```diff
@@ -61,9 +64,18 @@ export class Parser implements SubParser<Tx> {
     // failed to decode input data
     if (!decoded) return
 
+    let approveValue: string | null = null
+    let assetId: string | null = null
```

Comment:
IFFE here as well to remove let and simplify return payload

Reply:
Done in 338d2e7a9e8a1d522906358acb0913dae03ac334 / f91cd8c179d2b2ef0ca004c96bc16b9f18332464